entry_points.txt

```diff
@@ -39,6 +40,9 @@ cloud_providers = cfme.cloud.provider:CloudProviderCollection
 cloud_stacks = cfme.cloud.stack:StackCollection
 cloud_tenants = cfme.cloud.tenant:TenantCollection
 clusters = cfme.infrastructure.cluster:ClusterCollection
+config_manager = cfme.infrastructure.config_management:ConfigManagersCollection
+config_profile = cfme.infrastructure.config_management:ConfigProfilesCollection
+config_system = cfme.infrastructure.config_management:ConfigSystemsCollection
```

Comment:
All collections need to be plural like the above lines 40, 41, 42
config_managers = cfme.infrastructure.config_management:ConfigManagersCollection
config_profiles = cfme.infrastructure.config_management:ConfigProfilesCollection
config_systems = cfme.infrastructure.config_management:ConfigSystemsCollection